config/algolia.json

```diff
@@ -1,39 +1,18 @@
 {
   "index_name": "prod_docs",
-  "start_urls": [
-    "https://buildkite.com/docs"
-  ],
-  "stop_urls": [
-    "agent/v2"
-  ],
+  "start_urls": ["https://buildkite.com/docs"],
+  "stop_urls": ["agent/v2"],
   "selectors": {
-    "lvl0": {
-      "selector": ".Docs__nav__section-container.current > .Docs__nav__section-heading",
-      "global": true,
-      "default_value": "Documentation"
-    },
-    "lvl1": ".Page h1",
-    "lvl2": ".Page h2",
-    "lvl3": ".Page h3",
-    "lvl4": ".Page h4, .Page th",
-    "lvl5": ".Page h5, .Page td:first-child",
-    "lvl6": ".Page h6",
-    "text": ".Page p, .Page li, .Page td"
+    "lvl0": ".Page article h1",
+    "lvl1": ".Page article h2",
+    "lvl2": ".Page article h3",
+    "lvl3": ".Page article h4, .Page article th",
+    "lvl4": ".Page article h5, .Page article td:first-child",
+    "lvl5": ".Page article h6",
+    "text": ".Page article p, .Page article li, .Page article td"
   },
-  "selectors_exclude": [
-    "[data-algolia-exclude]",
-    ".Docs__toc",
-    ".HomepageCategory"
-  ],
+  "selectors_exclude": ["[data-algolia-exclude]", ".Toc", ".Footer", ".Nav"],
   "custom_settings": {
-    "synonyms": [
-      [
-        "Pipeline.yml",
-        "Pipeline.yaml",
-        "config.yml",
-        "config.yaml"
-      ]
-
-    ]
+    "synonyms": [["Pipeline.yml", "Pipeline.yaml", "config.yml", "config.yaml"]]
```

Comment:
Huh, that's cool. Will keep an eye out for any other obvious terms to include here

Reply:
I believe these are also configurable directly inside Algolia.